ramls/instance-iteration.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 1.0
+title: Iterate instances
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: Instance iteration
+    content: Iterate instances by generating domain events for them
+
+types:
+  iterationJobParams: !include instance-iteration/iteration-job-params.json
+  iterationJob: !include instance-iteration/iteration-job.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  language: !include raml-util/traits/language.raml
+  validate: !include raml-util/traits/validation.raml
+
+/instance-storage/instances/iteration:
+  displayName: Iterate instances
+  post:
+    description: Submit an iteration job
+    body:
+      application/json:
+        description: Iteration job parameters
+        type: iterationJobParams
+    responses:
+      200:
```

Comment:
Given this creates a job, could it be more specific to use 201 to denote that a new resource has been created?

Reply:
Thanks, updated.